src/components/ConversationSettings/LobbySettings.vue

```diff
@@ -41,7 +41,7 @@
 				<div>
 					<label for="moderation_settings_lobby_timer_field">{{ t('spreed', 'Meeting start time') }}</label>
 				</div>
-				<NcDatetimePicker id="moderation_settings_lobby_timer_field"
+				<NcDateTimePicker id="moderation_settings_lobby_timer_field"
```

Comment:
Works fine there with hardcoded 'true':
![image](https://github.com/nextcloud/spreed/assets/93392545/5939fcd7-1f44-43eb-89f1-ca3b26445051)

Maybe your devtools don't sync that fast, because disabled param depends on server response, and it's quite fast. Try with network throttled